packages/runtime/client/rest/src/index.ts

```diff
@@ -100,7 +110,22 @@ export interface RestAPI {
     contractId: ContractId,
     range?: TransactionsRange
   ): Promise<Transactions.GetTransactionsForContractResponse>;
-  //   createTransactionForContract: Transactions.POST; // - https://docs.marlowe.iohk.io/api/create-transactions // TODO: Jamie, lets unify names
+  /**
+   * Create an unsigned transaction which applies inputs to a contract.
+   * @see {@link https://docs.marlowe.iohk.io/api/apply-inputs-to-contract}
+   */
+  applyInputsToContract(kwargs: {
+    contractId: ContractId;
+    changeAddress: AddressBech32;
+    usedAddresses?: AddressBech32[];
+    collateralUTxOs?: TxOutRef[];
+    invalidBefore?: string;
+    invalidHereafter?: string;
+    version?: "v1";
+    metadata?: { [label: string | number]: any };
+    tags?: { [tag: string]: any };
+    inputs: Input[];
+  }): Promise<Transactions.TransactionTextEnvelope>;
```

Comment:
@hrajchert Having the API consumer in mind I designed this interface. I'm curious what you think.

I think this is quite straight forward for someone invoking this. Only the `inputs: Input[]` field could arguably also be redesigned to be less obfuscated but I think that is a bigger project (but worth considering).

Reply:
To be consistent with other endpoints I would put it as its own type, probably named `ApplyInputsToContractRequest` or `ApplyInputsRequest`, and change `kwargs` (not sure if it stands for keyword arguments 🤔 ) for `request`

Reply:
Alright, no worries. I'll fix it. Thanks!

Yes, `kwargs` stands for "keyword arguments". I don't mind using `request`.

Reply:
Fixed!